components/scheduler/executors.go

```diff
@@ -99,6 +118,32 @@ func (e taskExecutor) executeSchedulerWaitTask(
 	})
 }
 
+func workflowListDiff(newList, oldList []*commonpb.WorkflowExecution) []*commonpb.WorkflowExecution {
```

Comment:
comment what direction the diff is going (it looks like: oldList minus newList, assuming both are sorted?)